lms/templates/view.html

```diff
@@ -97,7 +100,16 @@ <h5 class="test-name">
     </div>
     {% if is_manager and not shared_url %}
     <div id="popular-comments">
-      <div id="exercise-comments">
+      <div id="exercise-grade" class="{{ direction }}">
+        <h2>{{ _('ציון התרגיל') }}</h2>
+        <div class="btn-group-vertical" id="solution-grade" role="group" aria-label="grades radio toggle button group">
+          {% for grade_mark in grade_marks %}
+            <input type="radio" class="btn-check" name="grade-mark" id="btnradio{{ loop.index }}" autocomplete="off" value="{{ grade_mark.id }}" {%- if solution['grade_mark'] and solution['grade_mark']['name'] == grade_mark.name %}checked{% else %}{% if loop.index == 1 %}checked{% endif %}{% endif -%}>
+            <label class="btn btn-outline-{{ grade_mark.color }}" for="btnradio{{ loop.index }}">{{ grade_mark.name }} <i class="fa fa-{{ grade_mark.icon }}"></i></label>
```

Comment:
I think we should make `icon` optional and show it only if there is an icon to show.
Also, we should consider putting it at the beginning of the word:

![image](https://user-images.githubusercontent.com/1492939/135715863-53e82041-2caf-439b-8bc9-11234064f032.png)

Reply:
I'd put `| e` on the color just to be sure